CHANGELOG.md

```diff
@@ -1,5 +1,57 @@
 # Changelog
 
+## 7.0.0 - 2022-04-??
+
+Are you ready to launch a [Human Rating Program](https://haystackconf.com/2019/human-judgement/)?   Quepid is now finally able to support you and your fellow human judges in gathering explicit ratings on your search results.
+
+Since the dawn of Quepid, we've suffered from an _ahem_ sub optimal rating interface for gathering human judgements.
+
+The rating interface failed most best practices for human rating, including suffering from position bias by listing documents in the order the search engine returned them, only supporting a single rating per query/doc pair, requiring lots of mouse movement and clicking.
+All that, and the UI is a combination of that required for a Relevancy Engineer with that of a Human Judge.  
```

Comment:
> All that, and the UI is a combination of that required for a Relevancy Engineer with that of a Human Judge.  

This sentence is slightly "bumpy" to read. (I think) I get it but it might benefit from improvement.
Maybe: "All that, and the UI is a combination of features required for a Relevancy Engineer with those of a Human Judge."